src/lib/components/workflow/live-child-workflows-table.svelte

```diff
@@ -0,0 +1,72 @@
+<script lang="ts">
+  import { page } from '$app/stores';
+
+  import Link from '$lib/holocene/link.svelte';
+  import Pagination from '$lib/holocene/pagination.svelte';
+  import TableHeaderRow from '$lib/holocene/table/table-header-row.svelte';
+  import TableRow from '$lib/holocene/table/table-row.svelte';
+  import Table from '$lib/holocene/table/table.svelte';
+  import { translate } from '$lib/i18n/translate';
+  import type { WorkflowExecution } from '$lib/types/workflows';
+  import { routeForEventHistory } from '$lib/utilities/route-for';
+
+  import WorkflowStatus from '../workflow-status.svelte';
+
+  export let children: WorkflowExecution[] = [];
+
+  $: ({ namespace } = $page.params);
+</script>
+
+<Pagination
+  items={children}
+  let:visibleItems
+  aria-label={translate('workflows.child-workflows')}
+  pageSizeSelectLabel={translate('common.per-page')}
+  previousButtonLabel={translate('common.previous')}
+  nextButtonLabel={translate('common.next')}
+>
+  <div slot="pagination-top" />
+  <Table class="w-full">
+    <caption class="sr-only" slot="caption"
+      >{translate('workflows.child-workflows')}</caption
+    >
+    <TableHeaderRow slot="headers">
+      <th class="max-md:hidden">{translate('common.status')}</th>
+      <th class="max-lg:hidden">{translate('common.type')}</th>
+      <th>{translate('workflows.child-id')}</th>
+      <th>{translate('workflows.child-run-id')}</th>
+    </TableHeaderRow>
+    {#each visibleItems as child}
```

Comment:
What do you think about just using `ChildWorkflowsTable` here and instead refactoring it to only take `children` and `namespace` props. 

The children could be formatted in the parent component to something like this ⬇️ 

```suggestion
    {#each visibleItems as {status, name, namespace, workflow, run}}
```

That way if the any future UI changes for this table only need to happen in one place.

Reply:
Definitely not a blocker! Especially if we think this table might diverge from the other.